packages/ckb-indexer/src/type.ts

```diff
@@ -90,6 +90,16 @@ export declare type Terminator = (
   cell: Cell
 ) => TerminatorResult;
 
+export declare type GetTerminableCells = (
+  searchKey: SearchKey,
+  terminator?: Terminator,
+  searchKeyFilter?: SearchKeyFilter
+) => Promise<GetCellsResults>;
+
+export interface GetTerminatedCellable {
```

Comment:
```ts
export interface TerminableCellFetcher
```
this may be more appropriate, since this interface is describing a cell getter, but getter may confuse with JavaScript's getter, so I suggest to change to `TerminableCellFetcher`